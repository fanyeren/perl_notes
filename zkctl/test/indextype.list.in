#!/bin/bash
source $(dirname $0)/common

TYPE=index-type
node_clear $TYPE

./zkctl -w $TYPE add --$TYPE-id=dnews --data={}
./zkctl -w $TYPE add --$TYPE-id=wp --data={}
./zkctl -w $TYPE add --$TYPE-id=vip --data={}
./zkctl -w $TYPE add --$TYPE-id=se --data={}
echo "add $TYPE result=$?"
./zkctl -w $TYPE list | sort
echo "list $TYPE result=$?"
./zkctl -w $TYPE del --$TYPE-id=wp
echo "del $TYPE result=$?"
./zkctl -w $TYPE list | sort
