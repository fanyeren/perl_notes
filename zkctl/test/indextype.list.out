add index-type result=0
dnews
se
vip
wp
list index-type result=0
del index-type result=0
dnews
se
vip
